spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/statsd/StatsdProperties.java

```diff
@@ -68,6 +69,11 @@
 	 */
 	private Integer queueSize = Integer.MAX_VALUE;
 
+	/**
+	 * Enables or disables sending of unchanged meters to StatsD server
```

Comment:
We're trying to make sure descriptions are harmonized. Please look at other boolean flag descriptions in the same area.